src/yggdrasil/admin.go

```diff
@@ -104,6 +105,23 @@ func (a *admin) init(c *Core, listenaddr string) {
 			*out = []byte(a.printInfos([]admin_nodeInfo{info}))
 		}
 	})
+	a.addHandler("getAuthBoxPubs", nil, func(out *[]byte, _ ...string) {
```

Comment:
Same as `PeerBoxPubs` in nodeconfig - we might want to give this a slightly friendlier name.